### DISCUSSION TITLE
Script just closes without giving password

### DISCUSSION THREAD

**Discussion by ObviouslyN0tMe** [illsk1lls/ZipRipper#36] [Q&A]
after telling me: Using Default wordlist... Loaded
the script just closes, trying to open a 7z file, 675KB, amd radeon rx 6700 xt i did use the additional script for amd cards but i think opencl is not the problem as it says ENABLED at the top of the window

**Comment by illsk1lls** [comment]
You can try changing the address on line 1055 from

`https://download.amd.com/dir/bin/amdocl64.dll/64CB5B3Ed36000/amdocl64.dll`

to

`https://download.amd.com/dir/bin/amdocl64.dll/66A33495d36000/amdocl64.dll`

This is a newer OpenCL driver version from AMD

You can browse them all at [https://download.amd.com/dir/bin/amdocl64.dll/](url)

**Reply by Hopeepoh** [reply]
How to open dll file?

**Reply by illsk1lls** [reply]
Place the file in C:\Windows\System32 and try running the script again

**Comment by illsk1lls** [comment]
The script has been updated again for Radeon GPU support. LMK
